app/styles/partials/event-card.scss

```diff
@@ -51,7 +51,3 @@
     height: 130px;
   }
 }
-
-.order.card {
```

Comment:
use min-width instead of width, and please check if its used anywhere else then maybe create a new class or inline style would work